File: app/src/main/java/com/ladsoft/bakingapp/fragment/di/RecipeFragmentBuilder.java
package com.ladsoft.bakingapp.fragment.di;

import com.ladsoft.bakingapp.fragment.RecipeFragment;

import dagger.Module;
import dagger.android.ContributesAndroidInjector;

@Module
public abstract class RecipeFragmentBuilder {
    @ContributesAndroidInjector(modules = RecipeFragmentModule.class)
    abstract RecipeFragment bindFragment();
}

File: app/src/main/java/com/ladsoft/bakingapp/fragment/di/RecipeFragmentBuilder.kt
package com.ladsoft.bakingapp.fragment.di

import com.ladsoft.bakingapp.fragment.RecipeFragment
import dagger.Module
import dagger.android.ContributesAndroidInjector

@Module
abstract class RecipeFragmentBuilder {
    @ContributesAndroidInjector(modules = [RecipeFragmentModule::class])
    abstract fun bindFragment(): RecipeFragment
}